practice-app/server/src/routes/api.js

```diff
@@ -1,16 +1,19 @@
 const express = require("express");
+
+const api = express.Router();
```

Comment:
The line: "const api = express.Router();" appears multiple times in the code. The upper one should be deleted

Reply:
I changed it in [this commit](https://github.com/bounswe/bounswe2022group3/commit/f52dd420ca710d9cafa4e3a6137cff0fd60bc7c9) after merging my code with master. Thank you for the reminder!